aproxy.go

```diff
@@ -0,0 +1,404 @@
+//go:build linux
+
+package main
+
+import (
+	"bufio"
+	"encoding/binary"
+	"errors"
+	"flag"
+	"fmt"
+	"golang.org/x/crypto/cryptobyte"
```

Comment:
It's common to have import sections, with the first section being stdlib imports, then a blank line, then 3rd party ones like this. The [`goimports` tool](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) does this automatically.

Reply:
Changed, thanks for the tool!